test/test_ensemble_builder/test_ensemble.py

```diff
@@ -156,9 +156,12 @@ def testNBest(ensemble_backend, ensemble_nbest, max_models_on_disc, exp):
 
     assert len(sel_keys) == exp
 
+    # Sort by name, then score
+    # Num_run=2 and num_run=3 have same performance.
+    # Prefer run_3 over run_2 (more budget/advanced run)
```

Comment:
Could you please revert this behavior? The idea is to actually use the earliest produced model.

Reply:
What about considering the model which takes less fit time if two models have same performance? Is it possible to consider this in current design?

Reply:
Done! You want the first model so you have a simpler one?

Reply:
Yes, correct.

Reply:
> What about considering the model which takes less fit time if two models have same performance? Is it possible to consider this in current design?

That's a great question - I don't think it's really easy to support as the necessary data structures aren't available at the moment.